### ISSUE TITLE
unsupported GTA IV audio files (PS3, Xbox 360)

### ISSUE THREAD

**Issue by RetingencyPlan** [losnoco/vgmstream#661]
ok, so seeing as there's the GTA IV audio format that vgmstream supports i thought i'd add in some more incentives to flesh the whole thing out.

[unsupported_ivaud](https://1drv.ms/u/s!Asofkpy7aqyKsnpinu6sSv33psdC?e=OlGVC8l)
these files were extracted by hand from .rpf files through an hex editor since there is no program/script that i'm aware of that handles .rpf files from either the PS3 or Xbox 360 versions of the game (**Grand Theft Auto IV**).

as for the "unsupported" part, i tried to back it up by passing these files into test.exe (test.exe -o blabla.wav blabla.ivaud) and it couldn't recognize any of these ivaud files.

**Comment by bnnm** [comment]
Files use a blocked format that seems similar to GTA5's AWC. That one is a bizarre, buggy block format so if it's anything like that it may take a while to add.

**Comment by RetingencyPlan** [comment]
right. in any case, i should be removing the link i've posted above. are you OK with this?

**Comment by bnnm** [comment]
Ok, I got the files. I recommend you upload stuff somewhere that lasts though, since I don't always save files for open issues until I get around to fix them.

**Comment by RetingencyPlan** [comment]
ummm... thanks.
anyway, link has been removed and first post has been updated to reflect this.

**Comment by bnnm** [comment]
Added.
